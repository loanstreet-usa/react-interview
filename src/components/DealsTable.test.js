import DealsList from './DealsTable';
import React from 'react';
import { create } from 'react-test-renderer';

it('Renders', () => {
    const dealsMockState =
        {
            sortField: 'dealType',
            reverseSort: false,
            deals: [{
                id: 1,
                institution: 'LS Credit Union',
                dealSize: '1000000',
                dealType: 'Consumer Auto',
                isPublished: true,
            }
                ,
            {
                id: 2,
                institution: 'LS Credit Union',
                dealSize: '5000000',
                dealType: 'Real Estate',
                isPublished: false,
            }
            ]
        };

        const target = create(<DealsList {...dealsMockState} />)
        expect(target).toMatchSnapshot();
});