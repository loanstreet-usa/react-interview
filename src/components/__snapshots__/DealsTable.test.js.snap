// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<div>
  <table
    className="DealsTable"
  >
    <thead>
      <tr>
        <th
          className="DealsTable--headerCell"
        />
        <th
          className="DealsTable--headerCell"
          onClick={[Function]}
        >
          Institution
        </th>
        <th
          className="DealsTable--headerCell"
          onClick={[Function]}
        >
          <span>
            *
          </span>
          Deal Type
        </th>
        <th
          className="DealsTable--headerCell"
          onClick={[Function]}
        >
          Deal Size
        </th>
        <th
          className="DealsTable--headerCell"
          onClick={[Function]}
        >
          Is Published?
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        className="DealsTableRow"
      >
        <td
          className="DealsTableRow--cell"
          onClick={[Function]}
        >
          x
        </td>
        <td
          className="DealsTableRow--cell"
        >
          LS Credit Union
        </td>
        <td
          className="DealsTableRow--cell"
        >
          Consumer Auto
        </td>
        <td
          className="DealsTableRow--cell"
        >
          $1,000,000
        </td>
        <td
          className="DealsTableRow--cell"
        >
          Yes
        </td>
      </tr>
      <tr
        className="DealsTableRow"
      >
        <td
          className="DealsTableRow--cell"
          onClick={[Function]}
        >
          x
        </td>
        <td
          className="DealsTableRow--cell"
        >
          LS Credit Union
        </td>
        <td
          className="DealsTableRow--cell"
        >
          Real Estate
        </td>
        <td
          className="DealsTableRow--cell"
        >
          $5,000,000
        </td>
        <td
          className="DealsTableRow--cell"
        >
          No
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
